src/lib.rs

```diff
@@ -216,6 +218,11 @@ impl<'a, Writer: Write> PrinterBuilder<'a, Writer> {
         self
     }
 
+    pub fn little_endian_dump(mut self, little_endian_dump: bool) -> Self {
```

Comment:
Similar here, let's call this `endianness` and let it accept a `Endianness` enun argument?